File: wrapper-app/src/main/java/com/wrapper/app/infrastructure/dto/generator/ProstorijaDto.java
package com.wrapper.app.infrastructure.dto.generator;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class ProstorijaDto {

    private String id;
    private String oznaka;
    private String oznakaSistem;
    private String tip;
    private int kapacitet;
    private List<String> orgJedinica;
    private List<String> sekundarnaOrgJedinica;
    private String sekundarniTip;
    private List<String> odobreniPredmeti;
}

File: wrapper-app-front/src/app/dtos/ProstorijaDto.ts
import { OrganizacionaJedinicaDto } from "./OrganizacionaJedinicaDto";

export interface ProstorijaDto {
    id: string
    oznaka: string;
    tip: string;
    kapacitet: number;
    orgJedinica: OrganizacionaJedinicaDto[];
    sekundarniTip:string;
}